src/openrct2/drawing/Drawing.Sprite.cpp

```diff
@@ -361,6 +362,36 @@ bool gfx_load_csg()
     }
 }
 
+std::optional<rct_gx> gfx_load_gx(std::vector<uint8_t> buffer)
+{
+    try
+    {
+        OpenRCT2::MemoryStream istream(buffer.data(), buffer.size());
+        rct_gx gx;
+
+        gx.header = istream.ReadValue<rct_g1_header>();
+
+        // Read element headers
+        gx.elements.resize(gx.header.num_entries);
+        read_and_convert_gxdat(&istream, gx.header.num_entries, false, gx.elements.data());
+
+        // Read element data
+        gx.data = istream.ReadArray<uint8_t>(gx.header.total_size);
+
+        // Fix entry data offsets
+        for (uint32_t i = 0; i < gx.header.num_entries; i++)
```

Comment:
I would perform this within the ImageTable. Once my PR is merged that will handle this bit.

Reply:
Do you want me to move the code to ImageTable or remove it from the PR entirely?

Reply:
move it for now.